source/particle/world.cc

```diff
@@ -72,6 +56,8 @@ namespace aspect
                            const unsigned int max_part_per_cell,
                            const unsigned int weight)
     {
+      World<dim>::connect_to_signals(this->get_signals());
+
```

Comment:
This is, in principle, a change that's completely independent of the rest. Can you submit it separately to separate concerns?